packages/controls/css/lumino.css

```diff
@@ -63,23 +63,27 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  * We've scoped the rules so that they are consistent with exactly our code.
  */
 
-.jupyter-widgets.widget-tab > .p-TabBar {
+.jupyter-widgets.widget-tab > .p-TabBar,
+.jupyter-widgets.jupyter-widget-tab > .p-TabBar {
```

Comment:
Wondering whether we want to deprecate the previous use of `.widget`.

For reference Lumino added some `<DEPRECATED>` comments for the switch from `p-Widget` to `lm-Widget`: 

https://github.com/jupyterlab/lumino/blob/e29f3c7172d2a256046e7fc509d8bacbb9093aa5/packages/widgets/style/widget.css#L11

Reply:
I can do that.
I've never deprecated css before, so... don't really have better ideas.